zhmcclient/_session.py

```diff
@@ -75,6 +75,11 @@ def __init__(self, host, userid=None, password=None):
           password (:term:`string`):
             Password of the HMC user to be used.
 
+          session_id (:term:`string`):
+            An opaque string that provides a cryptographically
+            strong identifier of the API session (known as a session id)
+            under which this request is executed.
+
```

Comment:
DONE: Addressed my comment in an update to this PR (differently that proposed in my comment, above).